server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -88,15 +88,17 @@ static class Opts extends Help {
     boolean zapCompactors = false;
     @Parameter(names = "-sservers", description = "remove scan server locks")
     boolean zapScanServers = false;
+    @Parameter(names = "-gc", description = "remove gc server locks")
+    boolean zapGc = false;
+    @Parameter(names = "-monitor", description = "remove monitor server locks")
+    boolean zapMonitor = false;
```

Comment:
We really need to stop using single dash for options. This is the only utility that does that, and it's not the normal convention, which can lead to confusion. It would be better to use double dash for new options, and to fix the other ones. This is not a utility we typically expect a user to call directly, so the risk of changing it to conform to norms is low.

Reply:
For the new options I can use double dash.  

For the existing options maybe we could define them like `@Parameter(names = {"-tservers", "--tservers"}` in 2.1 and then drop the single dash option later.  When adding the double dash option may also be an opportunity to make some of the terminology used more consistent w/ the scripts.   That could be done in a follow on PR, would add noise to this change.